cmd/otelcol/config/collector/config.d.linux/receivers/mongodb.discovery.yaml

```diff
@@ -47,9 +44,25 @@
 #               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_username="<username>"` and
 #               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_password="<password>"` environment variables.
 #       - status: partial
+#         regexp: 'SSL handshake received but server is started without SSL support'
+#         log_record:
+#           append_pattern: true
+#           body: >-
+#             Make sure the target database has SSL enabled or set insecure using the
+#             `--set splunk.discovery.receivers.mongodb.config.tls::insecure="<boolean>"` command or the
+#             `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_tls_x3a__x3a_insecure="<boolean>"` environment variable.
```

Comment:
Does `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_tls_x3a__x3a_insecure` work? Did you test this?

Reply:
That is a good catch. It doesn't work. I wanted to mimic this not sure how I ended up with t hat. The command definitely works

https://github.com/signalfx/splunk-otel-collector/blob/main/internal/confmapprovider/discovery/bundle/bundle.d/receivers/postgresql.discovery.yaml.tmpl#L53

Reply:
I will change it too.